src/app/clusters/wifi-network-diagnostics-server/wifi-network-diagnostics-server.cpp

```diff
@@ -28,6 +28,7 @@
 #include <lib/core/Optional.h>
 #include <platform/DiagnosticDataProvider.h>
 #include <tracing/macros.h>
+#include <tracing/metric_event.h>
```

Comment:
macros should be sufficient.

Reply:
I don't understand this requirement. If you look at the MATTER_LOG_XYZ macros, using those macros requires you to include TracingStructs.h. This is equivalent. 

To repeat: Metrics are not the same as tracing as defined in the macros.h. Metrics are data logging as defined in the macros.h

Reply:
Please also see my comment here: https://github.com/project-chip/connectedhomeip/pull/32223/files#r1500369705.